admin-dev/themes/new-theme/js/components/form/include-tax-field-toggle.ts

```diff
@@ -33,9 +33,20 @@ export default class IncludeTaxFieldToggle {
 
   $taxInclusionInputs: JQuery;
 
-  constructor(reductionTypeSelector: string, taxInclusionInputs: string) {
+  currencySymbolSelect: string;
+
+  reductionTypeAmountSymbolSelector: string;
```

Comment:
```suggestion
  reductionAmountSymbolSelector: string;
```

The input with the symbol is only containing the amount not the type, so I think the naming could be simplified (same for the other variables and component mappers) This would make it more understandable and shorter for a bit less code

Reply:
Fixed